wtforms_appengine/ndb.py

```diff
@@ -358,6 +360,9 @@ def convert_KeyProperty(self, model, prop, kwargs):
         else:
             return KeyPropertyField(**kwargs)
 
+    def convert__ClassKeyProperty(self, model, prop, kwargs):
```

Comment:
Yes I'm aware of that. I don't currently have pypi privileges, so can't help you there. I recently asked and am awaiting privileges, so maybe that will change. But for now, my comment above still stands.